benchmarks/benchmarks/bench_io.py

```diff
@@ -67,6 +67,17 @@ def time_vb_savez_squares(self):
         np.savez('tmp.npz', **self.squares)
 
 
+class LoadNpyOverhead(Benchmark):
+    # inspired by https://github.com/divideconcept/fastnumpyio
+    
+    def setup(self):
+        self.buffer = BytesIO()
+        np.save(self.buffer, get_squares_()['float32'])
+
+    def time_loadnpy_overhead(self):
+        self.buffer.seek(0, SEEK_SET)
```

Comment:
Setup is run every time anyway, so no need for seeking here.  Also `SEEK_SET` is the default anyway, I think?

Reply:
No, won't work. It is only run once while `time_loadnpy_overhead` is run multiple times to get the standard deviation. If the `seek` is removed

    python runtests.py --bench bench_io.LoadNpyOverhead.time_loadnpy_overhead

crashes with a traceback.

Reply:
hum, fine.  Really thought setup is run every time...

Reply:
I think it makes sense this way. Otherwise, why have a distinct setup function in the first place if one could just put everything into the other function?

Reply:
Because one is timed and the other is not, and because `setup` is used like that also in the testing frameworks IIRC.

Reply:
Okay, indeed, sounds valid, too.